input/blog/posts/_current-newsletter-draft.md

```diff
@@ -27,6 +27,12 @@ Welcome to the .NET Foundation update for March & April. Every other month, we'l
 Since releasing version 2.0 in coordination with the .NET 5 launch event at .NET Conf in November 2020, the exciting new modular application framework for Blazor has continued to mature and gain momentum. This release includes a variety of performance and user experience impovements for both users and developers. A total of 101 pull requests were merged from 9 different contributors, pushing the total number of project commits all-time over 1600. The 2.0.1 release is available for download on 
 [Github](https://github.com/oqtane/oqtane.framework/releases/tag/v2.0.1).
 
+### [NUnit](https://nunit.org)
+
+[NUnit Analyzers 3.1 and 2.1](https://github.com/nunit/nunit.analyzers/releases/tag/3.1.0) are available. The 3.1 version continues to enhance the experience of using NUnit with the C# 8 Nullable Reference Types feature.
```

Comment:
Should we include the blurb about why you would use 2.1 vs 3.1 for people that are new to the Analyzers?

>Version 3.0 and upwards works in Visual Studio 2019 and also enables supression of compiler errors such as errors arising from nullable reference types. For Visual Studio 2017 one must use versions below 3.0.

Reply:
Updated. I didn't mention nullable reference types since the previous line did, but what do you think?

Reply:
When we went with the two version solution, I had the expectation that more people were stuck on VS2017, but perhaps these do not use the analyzers. Comparing the download numbers on NuGet almost all people are using the 3.x version - 26,245 vs 117 downloads.

Reply:
There could be a correlation between the type of environment where people are using an IDE that's two years behind and the type of environment where people are not as likely to discover and opt in to a newish analyzer package.

Reply:
Or are not permitted to install anything not approved by their management. At least in my coaching experience, I saw a bit of that. As an historical note: Microsoft was at one time one of them.